__tests__/app.test.js

```diff
@@ -88,6 +81,25 @@ describe("/api/articles/:article_id", () => {
           });
         });
     });
+    test('should return the article that matches the given endpoint', () => {
+      return request(app)
+      .get('/api/articles/2')
+      .expect(200)
+      .then(({body}) => {
+        const testArticle = [ {
+          article_id: 2,
+          title: 'Sony Vaio; or, The Laptop',
+          topic: 'mitch',
+          author: 'icellusedkars',
+          body: 'Call me Mitchell. Some years ago—never mind how long precisely—having little or no money in my purse, and nothing particular to interest me on shore, I thought I would buy a laptop about a little and see the codey part of the world. It is a way I have of driving off the spleen and regulating the circulation. Whenever I find myself growing grim about the mouth; whenever it is a damp, drizzly November in my soul; whenever I find myself involuntarily pausing before coffin warehouses, and bringing up the rear of every funeral I meet; and especially whenever my hypos get such an upper hand of me, that it requires a strong moral principle to prevent me from deliberately stepping into the street, and methodically knocking people’s hats off—then, I account it high time to get to coding as soon as I can. This is my substitute for pistol and ball. With a philosophical flourish Cato throws himself upon his sword; I quietly take to the laptop. There is nothing surprising in this. If they but knew it, almost all men in their degree, some time or other, cherish very nearly the same feelings towards the the Vaio with me.',
+          created_at: '2020-10-16T05:03:00.000Z',
+          votes: 0,
+          article_img_url: 'https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700'
+        }]
+        expect(body.result[0].article_id).toBe(2)
+        expect(body.result).toEqual(testArticle)
+      })
+    });
   });
   describe("GET - error status", () => {
     test("error status: 400 - invalid article id", () => {
```

Comment:
Line 105, it says 400 when you are testing for a 404

Reply:
Please don't forget to update this description